assets/component/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: component
```

Comment:
folder should probably be named environment?

Reply:
agree and moved under environments directory.